evalml/tests/objective_tests/test_fraud_detection.py

```diff
@@ -32,7 +33,7 @@ def test_fraud_objective_function_amount_col(X_y_binary):
                           fraud_payout_percentage=.75,
                           amount_col="this column does not exist")
     y_predicted = pd.Series([.1, .5, .5])
-    y_true = [True, False, True]
+    y_true = pd.Series([True, False, True])
```

Comment:
Updated these to Series (per our docstrings) :d Alternative is to check for lists and handle 🤷

Reply:
Just filed https://github.com/alteryx/evalml/issues/1456 for this!